src/index.d.ts

```diff
@@ -979,6 +979,11 @@ declare namespace RamdaAdjunct {
          */
         neither(firstPredicate: Function, secondPredicate: Function): Function;
 
+        /**
+         * Returns `true` if both arguments are falsy, otherwise `false`.
```

Comment:
`Returns true if both arguments are false; false otherwise.` - just to make it more consistent with `ramda` `and` description

Reply:
👍